backends-common/rabbitmq/src/test/java/org/apache/james/backend/rabbitmq/RabbitMQFixture.java

```diff
@@ -37,6 +38,7 @@
     public static final String DIRECT = "direct";
     public static final boolean EXCLUSIVE = true;
     public static final boolean AUTO_DELETE = true;
+    public static final ImmutableMap<String, Object> NO_ARGUMENTS = ImmutableMap.of();
```

Comment:
Most of the contant of this class are not related to tests and deserves to be moved to main `Constant.java`